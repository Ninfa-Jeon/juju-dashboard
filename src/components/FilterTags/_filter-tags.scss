.p-filter-tags {
  padding: 0.75rem;

  &__input {
    font-size: 0.875rem;
    margin: 0;
    padding: 0.25rem 0.5rem;
  }
}
