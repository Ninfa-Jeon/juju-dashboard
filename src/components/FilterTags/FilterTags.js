import React from "react";

import "./_filter-tags.scss";

const FilterTags = () => {
  return (
    <div className="p-filter-tags">
      <form action="">
        <input
          type="text"
          placeholder="Filter terms"
          className="p-filter-tags__input"
        />
      </form>
    </div>
  );
};

export default FilterTags;
