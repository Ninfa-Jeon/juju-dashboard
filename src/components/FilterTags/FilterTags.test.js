import React from "react";
import { shallow } from "enzyme";

import FilterTags from "./FilterTags";

describe("FilterTags", () => {
  it("renders without crashing and matches snapshot", () => {
    const wrapper = shallow(<FilterTags />);
    expect(wrapper).toMatchSnapshot();
  });
});
