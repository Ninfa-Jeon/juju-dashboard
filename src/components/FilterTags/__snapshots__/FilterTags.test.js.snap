// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterTags renders without crashing and matches snapshot 1`] = `
<div
  className="p-filter-tags"
>
  <form
    action=""
  >
    <input
      className="p-filter-tags__input"
      placeholder="Filter terms"
      type="text"
    />
  </form>
</div>
`;
